src/Components/NavbarComponent/NewPost.js

```diff
@@ -1,7 +1,23 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { connect } from "react-redux";
+import { onPostArticleClicked, loadInitArticleDetail } from "../../ReduxStore/FeedDetails/feedActions";
 
 const InternalNewPost = props => {
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [content, setContent] = useState("");
+  const [tags, setTags] = useState("");
+
+// 从哪里拿到这篇文章的slug， 应该是内存
```

Comment:
This component is to create a new article that doesn't exist yet. So before the POST api call, these is no slug.

Reply:
Also if in Article detail component, we should check if the current user name matches the author before display edit and delete button